.github/workflows/pytorch-version-tests.yml

```diff
@@ -14,9 +14,9 @@ jobs:
       max-parallel: 10
       fail-fast: false
       matrix:
-        python-version: [3.7, 3.8, 3.9]
+        python-version: [3.7, 3.8, 3.9, 3.10]
         pytorch-version:
-          [1.11.0, 1.10.0, 1.9.1, 1.8.1, 1.7.1, 1.6.0, 1.5.1, 1.4.0, 1.3.1]
+          [1.12.1, 1.11.0, 1.10.0, 1.9.1, 1.8.1, 1.7.1, 1.6.0, 1.5.1, 1.4.0, 1.3.1]
```

Comment:
As it was discussed today, if you set 3.10 here, it will produce a matrix like
- 3.10 and 1.3.1
- 3.10 and 1.4.0
etc
However, those versions do not have any 3.10 support